docs/source/reference/modding/extended.rst

```diff
@@ -0,0 +1,208 @@
+In OpenMW it is possible to override animations via the same file for all actors which use a given basic animation file.
+
+If you want to override animations for all bipedal actors (they use the xbase_anim.nif skeleton), you can put your animations to the
+``Animations/xbase_anim`` folder in your ``Data Files`` (you can use the "multiple data folders" feature here).
+In this case any bipedal actor without custom animation will use your animations, but if he has additional animations, they have the higher priority.
+
+For example, all bipedal actors by default use the same spellcasting animations, so the xbase_anim override is enough.
+If you want to override walking animations, you should override the ``xbase_anim_female`` and ``xbase_anim_kna`` for females and beast races as well,
+because they have own walking animations, so they override ones which come from the xbase_anim (including its overrides from ``Animations`` folder).
+
+To enable this feature, you should have this line in your settings.cfg:
+
+``[Game]``
+
+``use additional anim sources = true``
+
+An example of mod which uses this feature is the `Almalexia's Cast for Beasts`_.
+
+
+Weapon sheathing support
+------------------------
+
+In OpenMW it is possible to show sheathing weapons on actors back, including quivers and scabbards.
+
+This feature conflicts with old mods which use a scripted scabbards, arrows with particles or decorative quivers (attached to the left pauldron, for example).
+
+1. Basics
+
+The minimum you need is the ``xbase_anim_sh.nif`` file from the `Weapon Sheathing`_ mod and this line in your settings.cfg:
+
+``[Game]``
+
+``weapon sheathing = true``
+
+The ``xbase_anim_sh.nif`` contains default placement points for different weapon types.
+It is enough to get the Gothic-style weapon sheathing for all bipedal actors (without quivers and scabbards).
+
+2. Scabbards
+
+To show a scabbard, you need a mesh with ``_sh`` suffix. For example, if a weapon uses the ``foo.sh`` mesh, a sheat mesh should have the ``foo_sh.nif`` name.
+There should be an least two nodes in the sheath file:
+
+``Bip01 Weapon`` - represents a weapon itself (may be just a grip for sword, for example). Is not shown when the weapon is drawn.
+
+``Bip01 Sheath`` - represents scabbards, quivers, etc. It is shown always when the weapon is equipped.
+
+You can move or rotate nodes if the default placing from the ``xbase_anim_sh.nif`` does not look good for your weapon.
+
+If you want to do not use this feature for a specific weapon, you can create a stub sheath mesh with just one root node.
+
+If you want to use the common weapon mesh, but use the custom placing, you can create a sheath mesh with an empty ``Bip01 Weapon`` node and move it as you want.
+
+3. Quivers
+
+To show the quiver for ranged weapon, you need these nodes in the sheath file:
+
+``Bip01 Sheath`` node, as for scabbards
+
+``Bip01 Ammo`` node to show ammo in the quiver
```

Comment:
Here and after ammo should be spelled as "ammunition" except the node naming instances.